package.json

```diff
@@ -116,5 +116,16 @@
       }
     ]
   },
-  "runkitExampleFilename": "example.js"
+  "runkitExampleFilename": "example.js",
+  "release": {
+    "branches": [
+      "+([0-9]).x",
+      "main",
+      "next",
```

Comment:
Does `next` need any special config, like `beta`, or it is automatically picked up from the name?

Reply:
it's automatically picked up from the name